www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Next generation web applications with async PHP
+info: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/
+location: Berlin
```

Comment:
Should probably also include country code like other talks :+1:

Reply:
Added country code